rest_framework/fields.py

```diff
@@ -178,7 +204,7 @@ def __init__(self, read_only=False, write_only=False,
         self.required = required
         self.default = default
         self.source = source
-        self.initial = self.initial if (initial is empty) else initial
+        self.initial = initial
```

Comment:
overall there is no behavior change as precedence is given to kwargs over class attributes; since everybody is de-facto using kwargs.